content/en/docs/started/installing-kubeflow.md

```diff
@@ -343,19 +343,16 @@ try out the end-to-end Kubeflow Platform capabilities.
 Users may choose to install the manifests for a specific Kubeflow version by following the
 instructions in the `README` of the [`kubeflow/manifests`](https://github.com/kubeflow/manifests) repository.
 
-- [**Kubeflow 1.8:**](/docs/releases/kubeflow-1.8/)
+- [**Kubeflow 1.8.1:**](/docs/releases/kubeflow-1.8/)
```

Comment:
This does not need to say `1.8.1`, given this section is about listing all the releases for `1.8` (we also list the `v1.8-branch` under it).

Reply:
This is something i can revert then.